### ISSUE TITLE
[ng] datagrid action overflow throws an error on loading for AoT compiled app

### ISSUE THREAD

**Issue by jeeyun** [vmware/clarity#529]
**Select one ...**  (check one with "x")
```
[x] bug
[ ] feature request
[ ] enhancement
```

### Expected behavior

Should not throw any exception when loading a datagrid with action overflow menu

### Actual behavior

Upon loading an application that is AoT compiled, I get an exception in console:

```EXCEPTION: Expression has changed after it was checked. Previous value: 'false'. Current value: 'true'.```

However, the datagrid functions fine after that, and renders and works normally. This does not happen in JiT compiled application.

### Reproduction of behavior

https://plnkr.co/edit/bAYeDO 
This loads fine in plunkr.. but same code doesn't work for AoT.

### Environment details

* **Angular version:** 2.4.1

* **Clarity version:**  0.8.8

* **OS and version:** all

* **Browser:** all

**Comment by ryanpflem** [comment]
I'm also getting this exception, except I'm using JIT and the datagrid does not function after.
I'm on Clarity v. 0.8.8 and Angular v. 2.4.8

My datagrid:
```
<clr-datagrid *ngIf="claims">
  <clr-dg-column [clrDgField]="'claimNumber'">Claim No.</clr-dg-column>
  <clr-dg-column [clrDgField]="'claimantFirstName'">First Name</clr-dg-column>
  <clr-dg-column [clrDgField]="'claimantLastName'">Last Name</clr-dg-column>
  <clr-dg-column [clrDgField]="'_changed'">Last Adjusted</clr-dg-column>

  <clr-dg-row *clrDgItems="let claim of claims" [clrDgItem]="claim">
      <clr-dg-action-overflow>
        <button class="action-item" (click)="onSelect(claim)">View</button>
      </clr-dg-action-overflow>
    <clr-dg-cell>{{claim?.claimNumber}}</clr-dg-cell>
    <clr-dg-cell>{{claim?.claimantFirstName}}</clr-dg-cell>
    <clr-dg-cell>{{claim?.claimantLastName}}</clr-dg-cell>
    <clr-dg-cell>{{claim?._changed | date:'shortDate'}}</clr-dg-cell>
  </clr-dg-row>

  <clr-dg-footer>
    {{pagination.firstItem + 1}} - {{pagination.lastItem + 1}} of {{pagination.totalItems}} Claims
    <clr-dg-pagination #pagination [clrDgPageSize]="10"></clr-dg-pagination>
  </clr-dg-footer>
</clr-datagrid>
```

Console error:

`EXCEPTION: Error in ./Datagrid class Datagrid - inline template:19:76 caused by: Expression has changed after it was checked. Previous value: 'false'. Current value: 'true'.`

**Comment by Shijir** [comment]
Hi @ryanpflem, could you try to reproduce your error using Plunkr, https://plnkr.co/8TwwdL if you are receiving the error in JIT mode?

**Comment by jbgarr** [comment]
I'm still running into this issue while running Clarity version 0.8.14. I am seeing this issue when trying to dynamically update the `clrDgPageSize` value of my datagrid. This was not an issue before I converted my app to use the Angular CLI build. Is there a suggested approach to allow users to select a new `clrDgPageSize` on the fly? I've provided a select element with its value bound to the same value I'm passing to `clrDgPageSize` and the first time I manually update the select element's value I get errors in the console and the datagrid is unresponsive after that (paging, sorting, filtering all fail).

Additional info:
I am using server-driven data

I would love to get this figured out and appreciate any help on this.
